addons/point_of_sale/models/pos_session.py

```diff
@@ -253,7 +254,7 @@ def login(self):
     def action_pos_session_open(self):
         # second browse because we need to refetch the data from the DB for cash_register_id
         # we only open sessions that haven't already been opened
-        for session in self.filtered(lambda session: session.state in ('new_session', 'opening_control')):
+        for session in self.filtered(lambda session: session.state == 'opening_control'):
```

Comment:
why did you changed that? Bellow you are writing the state opening_control on the session, but with the new filter, it is already in this state.

Reply:
`new_session` state was a leftover of Robin (?) previous code. It doesn't exist anymore, he didn't fully removed it from the code